tests/unit/VCRStorageFileTest.php

```diff
@@ -118,7 +118,7 @@ public function testRemove($path): void
 	public function testCreateSignedUrl($path, $expires, $options): void
 	{
 		\VCR\VCR::turnOn();
-		\VCR\VCR::insertCassette('storageFileTest');
+		\VCR\VCR::insertCassette('unittest_storage_file_test');
```

Comment:
How about we shorten this from `unittest_storage_file_test` to unit_storage_file_test. Let's do this for all instances.

Reply:
Done